pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -692,7 +693,7 @@ func (npm *nominator) DeleteNominatedPodIfExists(pod *v1.Pod) {
 // This is called during the preemption process after a node is nominated to run
 // the pod. We update the structure before sending a request to update the pod
 // object to avoid races with the following scheduling cycles.
-func (npm *nominator) AddNominatedPod(pi *framework.PodInfo, nodeName string) {
+func (npm *nominator) AddNominatedPod(pi *framework.PodInfo, nodeName *string) {
```

Comment:
Document the semantics of `nil` vs empty string.

Reply:
I updated to compare nil vs. non-nil. PTAL.